pandas/tests/plotting/test_converter.py

```diff
@@ -66,9 +66,16 @@ def test_registering_no_warning(self):
 
         # Set to the "warn" state, in case this isn't the first test run
         register_matplotlib_converters()
-        with tm.assert_produces_warning(DeprecationWarning, check_stacklevel=False):
-            # GH#30588 DeprecationWarning from 2D indexing
+
+        import matplotlib as mpl
+
+        version = mpl.__version__
+        if version > "3.1.2":
```

Comment:
use LooseVersion

Reply:
can also just define a new mpl skip decorator and skip if < 3.1.2

Reply:
thanks for your quick reply, changed, and let me know if this is what you meant @jreback